### ISSUE TITLE
Реализовать работу со значениями характеристик (CharacteristicValue)

### ISSUE THREAD

**Issue by ocassio** [ncedu-tlt/console-catalog#4]

**Comment by lordever** [comment]
Пытаюсь реализовать добавление характеристики, метод отрабатывает, но json файлик так и не создаёт. Пытаюсь протестировать категории, но там также json файлика нет, хотя методы отрабатывают без ошибок и в коде ошибок вроде тоже нет. Может я что то не правильно делал с Maven. Установку сделал с помощью комманды package в mvn.

**Comment by lordever** [comment]
хотя метод viewCategories также отрабатывает и читает из Json. Просто я сам файлик не могу найти.

**Comment by jegius** [comment]
Json в out падает после сохранения

**Comment by alzv0411** [comment]
Саша, ты решил проблему? Нашел файлы в папке out?

**Comment by lordever** [comment]
да. Проблема решена. Извините, что не ответил, было много работы. Другая проблема с редактированием характеристики: во время определения в классе переменной типа characteristicValue он её не видит. Бьюсь над этой проблемой уже 2 часа и не могу понять почему

**Comment by jegius** [comment]
У тебя там сравнение вместо неравенства стоит, вот и игнорируется ид для редактирования

**Comment by lordever** [comment]
Спасибо! Очень помог с этой проблемой

**Comment by lordever** [comment]
Все классы реализованы в полной мере, но есть недоработки. В частности добавлении и выводе значений характеристик. Буду продолжать работать, но можно считать эту часть выполненной?
